amundsen_application/api/utils/metadata_utils.py

```diff
@@ -183,9 +180,13 @@ def _convert_prog_descriptions(prog_descriptions: List = None) -> Dict:
                                 not in right_config, prog_descriptions))
             other.sort(key=lambda x: _sort_prog_descriptions(other_config, x))
 
-    updated_descriptions['left'] = left
-    updated_descriptions['right'] = right
-    updated_descriptions['other'] = other
+            updated_descriptions['left'] = left
```

Comment:
I have to check some edge cases in the JS code, but can we add a unit test to cover `prog_display_config = None`?

Reply:
yeap, will do

Reply:
I noticed that this implementation makes `left` and `right` optional on the dictionary, and verified that the JS code treats them optional as well so it shouldn't break anything.

However, when there is no configuration the behavior is just that the given `prog_descriptions` is simply not manipulated in any way. So what do you think of an alternative one line fix that just changes the initialization of `other` in the original implementation, so:
```
// line 160 of original implementation
other = prog_descriptions or [] # type: List
```

Reply:
@ttannis https://github.com/amundsen-io/amundsenfrontendlibrary/pull/672 with a updated test as well